### PR TITLE
paragraph on virtual environment added in the debiai installation

### PR DESCRIPTION
**Pull request by MacaireM**
paragraph on python virtual environment added in the debiai installation tutorial

### CODE REVIEW

File: documentation/introduction/gettingStarted/installation/README.md [hunk 1]
```diff
@@ -40,6 +40,31 @@ DebiAI offers several installation methods:
 
 - [Python](https://www.python.org/downloads/)
 
+Create a virtual python environment:
+
+```
+python -m venv name_of_virtual-environment
```
Comment (Tomansion) [inline documentation/introduction/gettingStarted/installation/README.md]: Change `name_of_virtual-environment` to an image name that would make more sense for a user to write

File: documentation/introduction/gettingStarted/installation/README.md [hunk 2]
```diff
@@ -40,6 +40,31 @@ DebiAI offers several installation methods:
...
+
+```
+name_of_virtual-environment\Scripts\Activate
+```
+
+Activate the virtual python environment created, on Windows terminal:
+
+```
+name_of_virtual-environment\Scripts\activate
+```
```
Comment (Tomansion) [inline documentation/introduction/gettingStarted/installation/README.md]: If the command is the same for Windows powershell and terminal, merge them in the documentation

File: documentation/introduction/gettingStarted/installation/README.md [hunk 3]
```diff
@@ -40,6 +40,31 @@ DebiAI offers several installation methods:
...
+python -m venv name_of_virtual-environment
+```
+
+Activate the virtual python environment created, on Unix and MacOS:
+
+```bash
+source name_of_virtual-environment/bin/activate
+```
+
+Activate the virtual python environment created, on Windows powershell:
```
Comment (Tomansion) [inline documentation/introduction/gettingStarted/installation/README.md]: powershell -> PowerShell

File: documentation/introduction/gettingStarted/installation/README.md [hunk 4]
```diff
@@ -40,6 +40,31 @@ DebiAI offers several installation methods:
 
 - [Python](https://www.python.org/downloads/)
 
+Create a virtual python environment:
+
+```
+python -m venv name_of_virtual-environment
+```
+
+Activate the virtual python environment created, on Unix and MacOS:
```
Comment (Tomansion) [inline documentation/introduction/gettingStarted/installation/README.md]: MacOS -> macOS

**Review by Tomansion — CHANGES_REQUESTED** [review]
Somme change suggestions

File: documentation/introduction/gettingStarted/installation/README.md (hunk 1 above)
Comment (MacaireM) [reply documentation/introduction/gettingStarted/installation/README.md]: :+1:

File: documentation/introduction/gettingStarted/installation/README.md (hunk 4 above)
Comment (MacaireM) [reply documentation/introduction/gettingStarted/installation/README.md]: :+1:

File: documentation/introduction/gettingStarted/installation/README.md (hunk 3 above)
Comment (MacaireM) [reply documentation/introduction/gettingStarted/installation/README.md]: :+1:

File: documentation/introduction/gettingStarted/installation/README.md (hunk 2 above)
Comment (MacaireM) [reply documentation/introduction/gettingStarted/installation/README.md]: There is a uppercase for PowerShell command and a lowercase.

Comment (Tomansion) [reply documentation/introduction/gettingStarted/installation/README.md]: Oh.. Ok, my bad

**Review by Tomansion — APPROVED** [review]

**Pull request merged**